graphs using dictionary

```diff
@@ -0,0 +1,32 @@
+thisdict = {
+    }
+    
+thisdict[1]=[[2,6],[3,7]]#first number = room number. second = distance of connection
+thisdict[2]=[[1,6]]
+thisdict[3]=[[1,7]]
+print(thisdict)
```

Comment:
Testing is executed in separate file - please remove and create test file

Reply:
^^I agree with felix

Reply:
The arrays ([2,6] & [3,7]) should be tuples by the way - as they'll be fixed values and only ever two of them.

Reply:
Surely the second element in e.g. [2,6] (so 6) isn't fixed? Because the distances between rooms changes with the weather?

Reply:
> Surely the second element in e.g. [2,6] (so 6) isn't fixed? Because the distances between rooms changes with the weather?
I thought the weather alteration was going to be done by a modifying variable